stdlib/task/src/main/java/org/ballerinalang/stdlib/task/actions/Pause.java

```diff
@@ -61,6 +65,7 @@ public void execute(Context context) {
         try {
             task.pause();
         } catch (SchedulingException e) {
+            LOG.error(e.getMessage(), e);
```

Comment:
Wny do we need to log this as we are returning the error message to the user?

Reply:
The returning error is a custom error we created inside ballerina native functions. But we are logging that error with the error / exception from the Quartz, for debugging purposes.

Reply:
Do we  follow this approach for any other stdlib extern functions? @wggihan Is this the best practice to follow?

Reply:
In here when we use the e.getMessage() it will return a custom error message that generated by us. The original quartz exception will be wrapped inside the SchedulingException. We don't want to expose the quartzness to the outside. But we might need that if there is an exception. That is the reason the log the exception. We have done that in other places as well.